PhotoKingdomAPI/PhotoKingdomAPI/Models/DesignModels/CountryPhotowarRequestedphotoUpload.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace PhotoKingdomAPI.Models
 {
-    public class CountryPhotowarRequestdphotoUpload
+    public class CountryPhotowarRequestedphotoUpload
```

Comment:
You modified my typo, but the project file [PhotoKingdomAPI.csproj](https://github.com/sofimofi/photokingdom-webapi/blob/design-model-associations-fix/PhotoKingdomAPI/PhotoKingdomAPI/PhotoKingdomAPI.csproj) didn't modified for the class.
Also, we have to modify ContinetPhotowarRequestedphotoUpload class same as this.
I'm sorry about this typo.